relationships.md

```diff
@@ -0,0 +1,336 @@
+    }
+  },
+  "_embedded": {
+    "relationshipType": {
+      "id": 1,
+      "leftLabel": "isAuthorOfPublication",
+      "rightLabel": "isPublicationOfAuthor",
+      "leftMinCardinality": 0,
+      "leftMaxCardinality": null,
+      "rightMinCardinality": 0,
+      "rightMaxCardinality": null,
+      "type": "relationshiptype",
+      "_links": {
+        "leftType": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1"
+        },
+        "rightType": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+        },
+        "self": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/1"
+        }
+      },
+      "_embedded": {
+        "leftType": {
+          "id": 1,
+          "label": "Publication",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1/relationshiptypes"
+            }
+          }
+        },
+        "rightType": {
+          "id": 2,
+          "label": "Person",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2/relationshiptypes"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The [relationship type](relationshiptypes.md) is embedded
+The 2 items are included as HAL links but are not embedded
+
+## Creating a relationship
+
+**POST /api/core/relationships?relationshipType=<:relationshipType>**
+
+A new relationship between 2 items can be created by specifying both items in a uri-list body and specifying the relationshipType as a parameter
+
+A sample CURL command would be:
+```
+curl -i -X POST 'https://dspace7-entities.atmire.com/rest/api/core/relationships?relationshipType=1' -H 'Authorization: Bearer eyJhbGciO…' -H "Content-Type:text/uri-list" --data 'https://dspace7-entities.atmire.com/rest/api/core/items/12623672-25a9-4df2-ab36-699c4c240c7e \n https://dspace7-entities.atmire.com/rest/api/core/items/5a3f7c7a-d3df-419c-8a2-f00ede62c60a'
+```
+
+The uri-list should always contain exactly 2 items. The first item will be used as the left Item. The second item will be used as the right Item.
+The relationshipType parameter is mandatory as well
+
+* 201 Created - if the operation succeed
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions
+* 422 Unprocessable Entity - if one of the items doesn't exist
+* 500 Internal Server Error - Other errors
```

Comment:
we usually don't specify the 500 error except if we have concrete examples to list (I found only an example in the browses.md). Not strictly required but I will prefer to have it removed

Reply:
I've removed it now